legend-engine-xt-mastery-grammar/src/main/java/org/finos/legend/engine/language/pure/dsl/mastery/compiler/toPureGraph/MasteryCompilerExtension.java

```diff
@@ -27,6 +27,8 @@
 
 import java.util.Collections;
 
+import static java.util.Objects.nonNull;
```

Comment:
There is a mix of using nonNull, isNull vs equality/non-equality checks with null. 
Could you not use nonNull, isNull so that this is consistent with the rest of the code base. 
Things like nonNull, isNull are predicates which make it nicer to write filters etc. I don't think they really help with normal conditionals.

Reply:
Thanks, @aziemchawdhary-gs , while I agree that we can replace `isNull()` and `nonNull` with equality checks, the methods may offer more readability and could be more expressive. 
However, I understand and respect the style and conventions in the codebase. I have pushed the changes as per your suggestion.